### ISSUE TITLE
Uploader v1.0.7 Does Not Start

### ISSUE THREAD

**Issue by lehitoskin** [Heroes-Profile/HeroesProfile.Uploader#10]
I updated to version 1.0.7 and when I ran the installer, a window kept popping up and closing itself and now my shortcut says this:
```
The item 'Heroesprofile.Uploader.exe' that this shortcut refers to has been changed or moved, so this shortcut will no longer work properly.

Do you want to delete this shortcut?
```

Re-re-installing does not change anything.

**Comment by lehitoskin** [comment]
Upon further inspection, it seems like the installer itself may be busted. It creates a folder Heroesprofile under `AppData\Local\`, but then inside there it creates the folder `app-1.0.6`, which is not the expected version number, so maybe it fails after that?

**Comment by lehitoskin** [comment]
Appears to have been resolved with v1.0.8
